pkgs/data/icons/nordzy-icon-theme/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenvNoCC
+, fetchFromGitHub
+, lib
+, bash
```

Comment:
Maybe this can be removed from args?

Reply:
Oh yeah, makes sense after the change to `patchShebangs`